blazeutils/spreadsheets.py

```diff
@@ -76,7 +76,6 @@ def xlsx_to_strio(xlsx_wb):
     """
         convert xlwt Workbook instance to a BytesIO instance
     """
-    _xlrd_required()
```

Comment:
I'm fine with removing it here, but then it needs to be added to xlsx_to_reader below (where xlrd is actually used). All of that should likely get marked as deprecated anyway - we can't keep using xlrd forever.